bananas/__init__.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (1, 0, 12, 'beta', 1)
+VERSION = (1, 0, 12, 'beta', 2)
```

Comment:
It's better to do version bumps in a separate non-PR commit to avoid merge-conflicts and have an explicit version-changing commit.